src/constructs/autoscaling/asg.ts

```diff
@@ -21,7 +21,6 @@ export interface GuAutoScalingGroupProps
       | "imageId"
       | "osType"
       | "machineImage"
-      | "instanceType"
```

Comment:
This is mandatory on AWS's `AutoScalingGroupProps`, so we no longer need to omit it.